idaes/apps/matopt/tests/test_matopt_objects_construction.py

```diff
@@ -195,229 +189,213 @@ def test_construct_CylindricalSector():
         np.array([0, 1, 0], dtype=float),
         np.array([0, 0, 1], dtype=float),
     )
-    return shape
 
 
 @pytest.mark.unit
-def test_construct_LinearTiling():
-    parallelepiped = test_construct_Parallelepiped()
-    cylinder = test_construct_Cylinder()
+def test_construct_LinearTiling(construct_Parallelepiped, construct_Cylinder):
+    parallelepiped = construct_Parallelepiped
+    cylinder = construct_Cylinder
     tiling = LinearTiling.fromParallelepiped(parallelepiped)
     tiling = LinearTiling.fromCylindricalShape(cylinder)
-    return tiling
 
 
 @pytest.mark.unit
-def test_construct_PlanarTiling():
-    shape = test_construct_Parallelepiped()
+def test_construct_PlanarTiling(construct_Parallelepiped):
+    shape = construct_Parallelepiped
     tiling = PlanarTiling(shape)
-    return tiling
 
 
 @pytest.mark.unit
-def test_construct_CubicTiling():
-    shape = test_construct_Parallelepiped()
+def test_construct_CubicTiling(construct_Parallelepiped):
+    shape = construct_Parallelepiped
     tiling = CubicTiling(shape)
-    return tiling
 
 
 @pytest.mark.unit
 def test_construct_ShiftFunc():
     transformation = ShiftFunc(np.array([1, 0, 0], dtype=float))
-    return transformation
 
 
 @pytest.mark.unit
 def test_construct_ScaleFunc():
     transformation = ScaleFunc(2.0)
-    return transformation
 
 
 @pytest.mark.unit
 def test_construct_RotateFunc():
     transformation = RotateFunc.fromXYZAngles(0.5, 0.5, 0.5)
-    return transformation
 
 
 @pytest.mark.unit
 def test_construct_ReflectFunc():
     transformation = ReflectFunc.acrossX()
-    return transformation
 
 
 @pytest.mark.unit
 def test_construct_Coef():
     coefficient = Coef([0.5, 0.5])
-    return coefficient
 
 
 @pytest.mark.unit
 def test_construct_LinearExpr():
     expression = LinearExpr()
-    return expression
 
 
-@pytest.mark.unit
-def test_construct_SumNeighborSites():
-    m = test_construct_MatOptModel()
+@pytest.fixture
+def construct_MatOptModel(construct_Canvas, construct_Atom):
+    canvas = construct_Canvas
+    a0, a1 = construct_Atom
+    atoms = [a0, a1]
+    model = MatOptModel(canvas, atoms)
+    return model
+
+
+@pytest.fixture
+def test_construct_SumNeighborSites(construct_MatOptModel):
```

Comment:
Please remove `test_` from the fixture name. Same comment for the next five fixture names.